react/components/molecules/key-values/key-values.js

```diff
@@ -15,6 +15,7 @@ export class KeyValues extends PureComponent {
                     value: PropTypes.string.isRequired,
                     keyColor: PropTypes.string,
                     valueColor: PropTypes.string,
+                    border: PropTypes.string,

```

Comment:
perhaps add to some items of the story the border property, to showcase the feature